kernel/src/sched.rs

```diff
@@ -240,34 +262,15 @@ impl Kernel {
     /// different index in the processes array. Note that a match _will_ be
     /// found if the process still exists in the correct location in the array
     /// but is in any "stopped" state.
+    #[inline(always)]
```

Comment:
This is still here?

Reply:
fixed. That did cost us a few dozen bytes